core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -351,13 +351,13 @@ class AdminClient(val time: Time,
   }
 
   def renewToken(hmac: ByteBuffer, renewTimePeriod: Long = -1): (Errors, Long) = {
-    val responseBody = sendAnyNode(ApiKeys.RENEW_DELEGATION_TOKEN, new RenewDelegationTokenRequest.Builder(hmac, renewTimePeriod))
+    val responseBody = sendAnyNode(ApiKeys.RENEW_DELEGATION_TOKEN, new RenewDelegationTokenRequest.Builder(new Array[Byte](hmac.remaining), renewTimePeriod))
```

Comment:
yes, I think we can delete those methods. since scala client is deprecated and this feature is  released recently, hope it wont create any compatibility issues.